### PR TITLE
feat: added Bitcoin address generation

### PR DESCRIPTION
**Pull request by xlanor**
**Description**

Added Bitcoin address generation based on discussion in #78 

**Are you trying to fix an existing issue?**

#78 

**Go Version**

```
abc@c63bccef3747:~/workspace/oss/faker$ go version
go version go1.17.3 linux/amd64
```

**Go Tests**

```
abc@c63bccef3747:~/workspace/oss/faker$ go test
PASS
ok      github.com/jaswdr/faker 5.644s
abc@c63bccef3747:~/workspace/oss/faker$ 
```

### CODE REVIEW

File: crypto.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	if dec == 0 {
+		return 48, 57
+	} else if dec == 1 {
+		return 65, 90
+	} else {
+		return 97, 122
+	}
+}
+
+// Helper function to return a bitcoin
```
Comment (jaswdr) [inline crypto.go:45]: just to confirm, by `return a bitcoin` you mean one bitcoin hash? if so, can you change this message to be clear that it is a bitcoin hash?

**Review by jaswdr — CHANGES_REQUESTED** [review]
@xlanor thanks for the PR, I have just some comments mostly related to comments and syntax, can you take a look when possible?

**Comment by xlanor** [comment]
Changes made as reuqested!

**Review by jaswdr — APPROVED** [review]

**Comment by jaswdr** [comment]
LGTM, merged 👍

**Pull request merged**